pyteal/ast/router.py

```diff
@@ -831,9 +844,10 @@ def add_method_handler(
         """
         if not isinstance(method_call, ABIReturnSubroutine):
             raise TealInputError(
-                "for adding method handler, must be ABIReturnSubroutine"
+                "for adding method handler, must be ABIReturnSubroutine but method_call is {type(method_call)}"
```

Comment:
you may want f string?
```suggestion
                f"for adding method handler, must be ABIReturnSubroutine but method_call is {type(method_call)}"
```